__tests__/app.test.js

```diff
@@ -235,3 +235,48 @@ describe.only("GET /api/articles/:article_id/comments - new comment", () => {
       });
   });
 });
+
+describe.only("PATCH /api/articles/:article_id", () => {
+  test("201: Update the votes for an article", () => {
+    const updatedVote = { inc_votes: 13 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(updatedVote)
+      .expect(201)
+      .then(({ body: { article } }) => {
+        expect(article[0].votes).toBeGreaterThan(0);
+        expect(article[0].article_id).toBe(1);
+        expect(article[0].votes).toBe(113);
```

Comment:
return rows[0] in your model then you should just get back an object here rather than an array with a single object in it!